fftools/ffmpeg_qsv.c

```diff
@@ -48,6 +48,21 @@ static int qsv_device_init(InputStream *ist)
 {
     int err;
     AVDictionary *dict = NULL;
+    HWDevice *hw_device = NULL;
+
+    if (ist->hwaccel_device) {
+        hw_device = hw_device_get_by_name(ist->hwaccel_device);
+        if (hw_device && hw_device->type == AV_HWDEVICE_TYPE_QSV) {
```

Comment:
maye you can use
if (hw_device) {
 if (hw_device->type != ..)


}

Reply:
change it, thanks